hoodie-common/pom.xml

```diff
@@ -50,6 +50,16 @@
             <plugin>
                 <groupId>org.apache.avro</groupId>
                 <artifactId>avro-maven-plugin</artifactId>
+                <configuration>
```

Comment:
have you tested everything with the hoodie cli, if you are actually replacing the json metadata with avro metadata. How/do we handle a mix of avro and json metadata?